ds_judgements_public_ui/sass/includes/_judgment_text.scss

```diff
@@ -114,7 +114,7 @@
     text-align: right;
 
     span {
-      font-weight: bolder;
+      font-weight: normal;
```

Comment:
I don't see a `white-space: nowrap`, have you commited and pushed those changes?